src/library-manager.js

```diff
@@ -161,8 +161,7 @@ class LibraryManager {
      * @returns {Promise<string[]>} the language codes for translations of this library
      */
     async listLanguages(library) {
-        return this._libraryStorage.getLanguageFiles(library);
-
+        return this._libraryStorage.getLanguageFiles(library).catch(error => [])
```

Comment:
I think we should settle on a style to handle promises for the project:

I prefer the async await style as it allows you to write code that look like synchronous code and you don't have lots of nested callbacks. If you don't use async await the changing identation often also confuses change comparison tools and it becomes more difficult to see what has actually been changed and what is simply new indentation because a function call has been added or removed somewhere. This was the case when merging in changes in index.js earlier in the project, and created the need for some longer manual comparison of changes.

Reply:
I am not too familiar with the async await style. How would that look like? Wrap it in a try catch block?

Reply:
In this case it would be:

```js
try {
  return await this._libraryStorage.getLanguageFiles(library);
{
catch {
  return [];
}
```

As it‘s only a one liner the benefit isn‘t that obvious, but in longer functions it becomes very visible.

Reply:
Alright. I see the advantage. I will change it as soon as possible.